experiment/autobumper/bumper/bumper.go

```diff
@@ -261,3 +534,37 @@ func generatePRBody(images map[string]string, assignment string) string {
 	testImagesSummary := generateSummary("test-image", testImageRepo, testImagePrefix, false, images)
 	return prowSummary + "\n\n" + testImagesSummary + "\n\n" + assignment + "\n"
 }
+
+func getAssignment(oncallAddress string) string {
+	if oncallAddress == "" {
+		return ""
+	}
+	oncaller, err := getOncaller(oncallAddress)
+	if err == nil {
+		if oncaller != "" {
+			return "/cc @" + oncaller
+		}
+		return "Nobody is currently oncall, so falling back to Blunderbuss."
+	}
+	return fmt.Sprintf("An error occurred while finding an assignee: `%s`.\nFalling back to Blunderbuss.", err)
+}
+
+func getOncaller(oncallAddress string) (string, error) {
+	req, err := http.Get(oncallAddress)
+	if err != nil {
+		return "", err
+	}
+	defer req.Body.Close()
+	if req.StatusCode != http.StatusOK {
+		return "", fmt.Errorf("HTTP error %d (%q) fetching current oncaller", req.StatusCode, req.Status)
+	}
+	oncall := struct {
+		Oncall struct {
+			TestInfra string `json:"testinfra"`
+		} `json:"Oncall"`
+	}{}
+	if err := json.NewDecoder(req.Body).Decode(&oncall); err != nil {
+		return "", err
```

Comment:
Please disambiguate errors when functions return an error from multiple locations.

```golang
return fmt.Errorf("get: %v", err)
return fmt.Errorf("decode response: %v", err)
```

(please apply this everywhere you are returning err directly, not just in this function)

Reply:
Made the change in everywhere it's needed.